playground/frontend/lib/modules/examples/repositories/models/get_list_of_examples_request.dart

```diff
@@ -23,4 +23,14 @@ class GetListOfExamplesRequestWrapper {
   final String? category;
 
   GetListOfExamplesRequestWrapper({required this.sdk, required this.category});
+
+  @override
+  bool operator ==(Object other) =>
+      identical(this, other) ||
+          other is GetListOfExamplesRequestWrapper &&
+              category == other.category &&
+              sdk == other.sdk;
+
+  @override
+  int get hashCode => category.hashCode ^ sdk.hashCode;
```

Comment:
Nit: Flutter has the [hashValues](https://api.flutter.dev/flutter/dart-ui/hashValues.html) function which is more standard.

Reply:
Hello, @KonradJanica! Thank you for the review, fixed that in the last commit.